src/main/java/net/rubygrapefruit/platform/file/FileWatcher.java

```diff
@@ -0,0 +1,20 @@
+package net.rubygrapefruit.platform.file;
+
+import java.io.Closeable;
+import java.io.IOException;
+
+/**
+ * A handle for watching file system locations.
+ */
+public interface FileWatcher extends Closeable {
```

Comment:
Shouldn't this be called `FileWatch`?

Reply:
I don't feel strongly either way. Arguments?

Reply:
I'd leave it as `FileWatcher` actually. "Watch" as a noun reminds me of the Night's Watch in GoT. 

![](https://s1.r29static.com/bin/entry/e47/720x864,85/1801540/image.webp)